kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue

```diff
@@ -312,8 +312,8 @@
       cancelStyleOverrides() {
         return {
           color: this.$themeTokens.textInverted,
-          'background-color': this.$themePalette.red.v_700,
-          ':hover': { 'background-color': this.$themePalette.red.v_900 },
+          'background-color': this.$themePalette.red.v_1100,
+          ':hover': { 'background-color': '#A81700' },
```

Comment:
Is there a particular reason a hardcoded color code is used here?

Reply:
Ah yes, the reason was that we can't use lighter red than `palette.red.v_1100` to ensure sufficient contrast, and at the same time we don't have darker red in the palette available. I discussed this with designers and they recommended to use this hardcoded color for this particular occasion (with that we may want to add it to the palette if we need it more times). I think in any case I should add a comment because it doesn't follow common pattern.

Reply:
Perhaps the way to do it would be to use a snippet like this? https://css-tricks.com/snippets/javascript/lighten-darken-color/ and then we programmatically darken/lighten a specific theme palette colour rather than hard coding the result?

Reply:
I experimented with it and it works well, great idea. I imagine it as a KDS utility for which I'd like to add a bit of guidance on the Colors page. Designers had some thinking in progress about how to allow for colors outside of the palette so I will confirm with them at first.

Reply:
I got confirmation from designers and we will do this, but I will work on it as follow-up since we need to clarify some things.